src/driver/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ add_custom_command(
 set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/driver)
 rocm_clang_tidy_check(driver)
 
-target_link_libraries(driver migraphx_all_targets migraphx_onnx migraphx_tf)
+target_link_libraries(driver migraphx_all_targets migraphx_onnx migraphx_tf migraphx_py)
```

Comment:
`migraphx_py` library is the entry point to use the API but its just a shim layer which will forward to `migraphx_py_${PYTHON_VERSION}` depending on which python version is installed. That is why its called `py_loader.cpp` since it will just load the function in `py.cpp` using the correct python version.